src/Microsoft.ML.Transforms/LambdaTransform.cs

```diff
@@ -3,7 +3,9 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
+using System.Diagnostics.Contracts;
```

Comment:
Just curious, why do we need `System.Diagnostics.Contracts` in here?

Reply:
Fixed. 

---
In reply to: [401444214](https://github.com/dotnet/machinelearning/pull/4989#discussion_r401444214) [](ancestors = 401444214)